security-group-variables.tf

```diff
@@ -3,8 +3,17 @@
 
 variable "create_security_group" {
   type        = bool
-  default     = true
-  description = "Set `true` to create and configure a Security Group for the cluster."
+  default     = false
```

Comment:
Why is this disabled ?

Reply:
Oh I see it in the PR description.

Reply:
the default should be `false` to not create a new SG.
We don't need to make it backwards compatible by using a default that is not suitable for the module.
People will run `terraform plan`, see that the SG will not be created or Tf will try to delete it, and they can set it to `true`.